ipa-multipoint/src/test/java/org/hyperledger/besu/nativelib/ipa_multipoint/CommitRootTest.java

```diff
@@ -0,0 +1,46 @@
+package org.hyperledger.besu.nativelib.ipa_multipoint;
+
```

Comment:
Seems good, could you just add a header like this file https://github.com/hyperledger/besu-native/blob/main/gnark/src/main/java/org/hyperledger/besu/nativelib/gnark/LibGnark.java#L1